PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/VideoInterstitialAdUnit.java

```diff
@@ -17,16 +17,24 @@
 package org.prebid.mobile;
 
 import androidx.annotation.NonNull;
+
 import org.prebid.mobile.api.data.AdFormat;
 import org.prebid.mobile.rendering.models.AdPosition;
 import org.prebid.mobile.rendering.models.PlacementType;
 
+import java.util.EnumSet;
+
+/**
+ * @deprecated Use InterstitialAdUnit constructor with adUnitFormats parameter:
+ * `EnumSet.of(AdUnitFormat.VIDEO);`
+ */
+@Deprecated
```

Comment:
Can we add a custom message to the compilation warning? It would be great to provide the info about the new approach right in the build log 

<img width="498" alt="image" src="https://user-images.githubusercontent.com/21169695/232062358-1d26c737-163b-4419-a169-5412b80aae6f.png">

Reply:
We can't add a custom message to the studio. To understand the problem users read the `@deprecated` tag in the documentation or use other static analyzers.